### ISSUE TITLE
Labels as validation signals to meeting planners are a source of confusion

### ISSUE THREAD

**Issue by ianbjacobs** [w3c/tpac-breakouts#40]
We now have experience that when validation tools use labels to communicate information to meeting planners, that is creating confusion for session chairs. 

@tidoust has mentioned variables as an alternative.

I expect we'll revisit this topic for TPAC 2024.

**Comment by tidoust** [comment]
Given that some proposers keep fighting to force removal of labels, I went ahead and implemented the change.

Validation errors, warnings and checks are now managed through dedicated custom fields at the project level, with dedicated project views to track them, see the [Errors](https://github.com/orgs/w3c/projects/29/views/4) view for instance.

Validation results still appear in the issue itself, but they are "hidden" under the project panel (same as for the room and slot). That's less convenient for meeting planners, but much less likely to pose any problem to session proposers.

I updated the instructions for meeting planners accordingly, also adding a short explanation of [possible validation results](https://github.com/w3c/tpac-breakouts/wiki/For-TPAC-Meeting-Planners#session-validation).

Only labels remaining on the repository are the "session" label and track labels.

(I don't guarantee that the new code does not have bugs. But then the older code wasn't bug free either ;))
